README.md

```diff
@@ -0,0 +1,49 @@
+# eLife Search
+
+This project is using elasticsearch to index content of the eLife articles and provide a full-text search for Journal.
+
+To reliably recreate any issue experienced in CI or Prod you should continue to use [builder](https://github.com/elifesciences/builder).
+
+## Docker for local development
+
+Important: Keep in mind that docker is just used to improve the developer experience.
+
+1. Clone the project `git clone https://github.com/elifesciences/search.git`
+2. Rename `/dev/config.php.dist` on local to `/dev/config.php`
+3. Run `mkdir logs && sudo chmod 777 ./logs`
+4. Run `docker-compose -f dev/docker-compose.yaml up --build`
+
+### Setup
+
```

Comment:
Might be worth adding a step to make sure that we are ready for setup? For instance, making sure that opensearch is running/working?